### DISCUSSION TITLE
Service is not reachable  - Django

### DISCUSSION THREAD

**Discussion by amalraj1264** [easypanel-io/community#99] [Q&A]
I've successfully deployed a Django application using Nixpacks and PostgreSQL, and all migrations have completed correctly. However, when I try to access the application, I receive an 'Service is not reachable' error

![image](https://github.com/user-attachments/assets/741ce67d-f2de-4a19-bd74-a9b8737fcd13)

![neww](https://github.com/user-attachments/assets/2a05b8e6-cd84-409d-89ad-859231175692)

![image](https://github.com/user-attachments/assets/ff0c9e9a-4b81-4cb0-b717-4010dec05033)
![new1](https://github.com/user-attachments/assets/ea0f51ce-b3c8-4bb1-a72f-52938b47937e)

Please help me resolve this issue.
Thanks

**Comment by deiucanta** [comment]
In the domains tab, you should set the internal port to 5432

**Comment by amalraj1264** [comment]
IT works now. Thanks for quick support. 

Is there a specific setting or configuration in EasyPanel that could cause a Django application to be accessible on port 5432 instead of the standard HTTP port?
Other applications (React, Wordpress) are using port 80 and Postgres's internal port is set to 5432.
